src/games/settings/GameSettingsModal.tsx

```diff
@@ -69,6 +69,12 @@ export function GameSettingsModal(props: IGameSettingsModalProps) {
             checked={settings?.noHighlight100PercentUsage}
             onChange={onChangeHandler('noHighlight100PercentUsage')}
           />
+          <Checkbox
+            label="Animate connections"
```

Comment:
Seems good, but if you want to have it as a positive-flag there should be a corresponding migration that ticks this.
Elseway we could change this to a "Disable connection animations" flag (negative) so that it's backward compatible. I'd prefer the latter